add-on/chrome/bootstrap.js

```diff
@@ -522,9 +537,13 @@ var WindowListener = {
           gBrowser.tabContainer.addEventListener("TabSelect", this);
           this._listeningToTabSelect = true;
 
+          titleChangedListener = this.handleDOMTitleChanged.bind(this);
+
           // Watch for title changes as opposed to location changes as more
           // metadata about the page is available when this event fires.
-          gBrowser.addEventListener("DOMTitleChanged", this);
+          this.mm.addMessageListener("loop@mozilla.org:DOMTitleChanged",
+            titleChangedListener);
```

Comment:
You 'could' put the assignment here as well: `titleChangedListener = this.handleDOMTitleChanged.bind(this));`... up to you.